docs/source/configuration/key_value_service_configs/oracle_key_value_service_config.rst

```diff
@@ -17,9 +17,9 @@ runtime dependency. In gradle this looks like:
 
 e.g.
 
-.. code-block:: groovy
+.. code-block-with-version-replacement:: groovy
 
-  runtime 'com.palantir.atlasdb:atlasdb-dbkvs:0.19.0'
+  runtime 'com.palantir.atlasdb:atlasdb-dbkvs:|last-release|'
```

Comment:
As discussed via chat, I've suggested using `latest` instead of `last_release`, to avoid having to remember where to use hyphen and underscore as delimiters.